### ISSUE TITLE
The variable LEM/COMMON/TIMER::*TIMER-MANAGER* is unbound.

### ISSUE THREAD

**Issue by tmdemelo** [lem-project/lem#609]
After #604 got fixed, another issue with special variables:

```
The variable LEM/COMMON/TIMER::*TIMER-MANAGER* is unbound.
   [Condition of type UNBOUND-VARIABLE]

[...]

Backtrace:
  0: (LEM/COMMON/TIMER::START-TIMER-THREAD #<LEM/COMMON/TIMER:TIMER (LAMBDA NIL :IN LEM-INTERFACE:DISPLAY-POPUP-MESSAGE) {10068A0C03}> 1000 NIL)
  1: ((:METHOD LEM/COMMON/TIMER:START-TIMER (LEM/COMMON/TIMER:TIMER T)) #<LEM/COMMON/TIMER:TIMER (LAMBDA NIL :IN LEM-INTERFACE:DISPLAY-POPUP-MESSAGE) {10068A0C03}> 1000 NIL) [fast-method]
  2: ((:METHOD LEM-INTERFACE:DISPLAY-POPUP-MESSAGE (T T)) #<unused argument> "hello world" :TIMEOUT 1 :DESTINATION-WINDOW NIL :SOURCE-WINDOW NIL :STYLE (:GRAVITY :FOLLOW-CURSOR)) [fast-method]
  3: ((:METHOD LEM:SHOW-MESSAGE (T)) "hello world" :TIMEOUT 1) [fast-method]
  4: (LEM:MESSAGE "hello world")
  5: (SB-INT:SIMPLE-EVAL-IN-LEXENV (LEM:MESSAGE "hello world") #<NULL-LEXENV>)
  6: (EVAL (LEM:MESSAGE "hello world"))
```

As you might notice from the backtrace, the issue happens at `lem/common/timer::start-timer-thread`. It affects calling `lem:message` and `lem:show-message` with a non-nil `:timeout` arg.

**Comment by cxxxr** [comment]
Thank you for your report.
Fixed in commit 37451c2742506443c8b16d12c751b9352f56f6a0.

**Comment by tmdemelo** [comment]
Thank you very much for the fix.
